app/models/organisation.rb

```diff
@@ -0,0 +1,3 @@
+class Organisation < ActiveRecord::Base
```

Comment:
Could you please add the two magic comments to the beginning of this file? Please refer to any other .rb file in the dev branch for an example.